7_deploy_osh/roles/suse-osh-deploy/tasks/main.yml

```diff
@@ -9,9 +9,8 @@
   tags:
     - always
 
-- name: Create ses_cluster_configuration
-  set_fact:
-    ses_cluster_configuration: "{{ lookup('file', suse_osh_deploy_ses_config_path) | from_yaml }}"
+- name: Get ses_cluster_configuration
+  include_vars: "{{ suse_osh_deploy_ses_config_path }}"
```

Comment:
This is wrong, there's no `ses_cluster_configuration` key in the default version of  `ses_config.yml` file. We might adapt this file's format, or we need to change this `include_vars` part